css/user_interface.css

```diff
@@ -6,6 +6,7 @@ body {
     margin: 0px;
     font-family: Verdana, sans-serif;
     background-color: #F4F4F4;
+    border: 1px solid black;
```

Comment:
I placed the border automaticlly with powerpoint, but maybe without border and a shadow it would look better (but I assume that it's harder to implement).